tests/wallet/dao_wallet/test_dao_wallets.py

```diff
@@ -0,0 +1,2067 @@
+            wallet_1,
+            treasury_id,
+        )
+    assert dao_wallet_1 is not None
+    assert dao_wallet_0.dao_info.treasury_id == dao_wallet_1.dao_info.treasury_id
+
+    # Get the cat wallets for wallet_1
+    cat_wallet_1 = dao_wallet_1.wallet_state_manager.wallets[dao_wallet_1.dao_info.cat_wallet_id]
+    assert cat_wallet_1
+
+    # Send some cats to the dao_cat lockup
+    dao_cat_amt = uint64(100)
+    async with wallet_node_0.wallet_state_manager.lock:
+        txs, new_coins = await dao_wallet_0.create_new_dao_cats(dao_cat_amt, push=True)
+    sb = txs[0].spend_bundle
+    await time_out_assert_not_none(5, full_node_api.full_node.mempool_manager.get_spendbundle, sb.name())
+    await full_node_api.process_transaction_records(records=txs)
+
+    # Give the full node a moment to catch up if there are no trusted peers
+    if not trusted:
+        await asyncio.sleep(1)
+    for i in range(1, num_blocks):
+        await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(puzzle_hash_0))
+
+    # Test that we can get spendable coins from both cat and dao_cat wallet
+    fake_proposal_id = Program.to("proposal_id").get_tree_hash()
+    spendable_coins = await dao_cat_wallet_0.wallet_state_manager.get_spendable_coins_for_wallet(
+        dao_cat_wallet_0.id(), None
+    )
+
+    assert len(spendable_coins) > 0
+    coins = await dao_cat_wallet_0.advanced_select_coins(1, fake_proposal_id)
+    assert len(coins) > 0
+    # check that we have selected the coin from dao_cat_wallet
+    assert list(coins)[0].coin.amount == dao_cat_amt
+
+    # send some cats from wallet_0 to wallet_1 so we can test voting
+    cat_txs = await cat_wallet_0.generate_signed_transactions([cat_amt], [ph_1])
+    await wallet.wallet_state_manager.add_pending_transaction(cat_txs[0])
+    sb = cat_txs[0].spend_bundle
+    await time_out_assert_not_none(5, full_node_api.full_node.mempool_manager.get_spendbundle, sb.name())
+    await full_node_api.process_transaction_records(records=cat_txs)
+
+    for i in range(1, num_blocks):
+        await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(puzzle_hash_0))
+
+    await time_out_assert(10, cat_wallet_1.get_confirmed_balance, cat_amt)
+
+    # Smaller tests of dao_wallet funcs for coverage
+    # adjust filter level
+    await dao_wallet_0.adjust_filter_level(uint64(10))
+    assert dao_wallet_0.dao_info.filter_below_vote_amount == uint64(10)
+
+    assert dao_wallet_0.puzzle_for_pk(G1Element()) == Program.to(0)
+    assert dao_wallet_0.puzzle_hash_for_pk(G1Element()) == Program.to(0).get_tree_hash()
+    assert (await dao_wallet_0.get_new_puzzle()) == Program.to(0)
+
+    await dao_wallet_0.set_name("Renamed Wallet")
+    assert dao_wallet_0.get_name() == "Renamed Wallet"
+
+    new_inner_puzhash = await dao_wallet_0.get_new_p2_inner_hash()
+    assert isinstance(new_inner_puzhash, bytes32)
+
+    # # generate new dao spends with bad cat details
```

Comment:
why is this commented out?

Reply:
We removed the functions for creating DAO's for existing CATs and creating DAOs 'offline' via spendbundle, since they were out of scope for the beta. These tests were relating to those functions, so I've just removed it for now. We've stored the unused code in a WIP branch so it can be picked up again later.